### PR TITLE
Chunking

### PR DESCRIPTION
**Pull request by Yadu-M**

### CODE REVIEW

**Comment by namoosir** [comment]
@Yadu-M fix merge conflicts and test the entire functionality again and then tag me

**Review by namoosir — CHANGES_REQUESTED** [review]
merge conflicts

File: functions/src/callable.ts [hunk 1]
```diff
@@ -1,14 +1,39 @@
...
+//     .replaceAll("https://", "")
+//     .replaceAll("http://", "")
+//     .replaceAll(".com", " dot com")
+//     .replaceAll(".org", " dot org")
+//     .replaceAll(".co.uk", " dot co dot UK")
+//     .replaceAll(/```[\s\S]*?```/g, "\nAs shown on the app.\n")
+//     .replaceAll(
+//       /([a-zA-Z0-9])\/([a-zA-Z0-9])/g,
+//       (_: any, precedingText: string, followingText: string) => {
+//         return precedingText + " forward slash " + followingText;
```
Comment (namoosir) [inline functions/src/callable.ts]: Delete all that. If we wanna look at it later we will check git history

**Review by namoosir — CHANGES_REQUESTED** [review]
Remove comments. Then just merge it yourself

File: functions/src/callable.ts (hunk 1 above)
Comment (Yadu-M) [reply functions/src/callable.ts]: aight

**Pull request merged**